src/api.rs

```diff
@@ -339,6 +339,9 @@ impl Config {
       aom_dsp_rtcd();
     }
 
+    let mut packet_data = Vec::new();
+    write_temporal_delimiter(&mut packet_data).unwrap();
```

Comment:
Please fail gracefully.

Reply:
This one is to something I just alloc'd so I know it can't fail, can I leave it?